source/native/symbulation_sgp.cc

```diff
@@ -71,28 +120,30 @@ int symbulation_main(int argc, char *argv[]) {
     }
   }
 
-  {
-    ofstream mutualism_file;
-    std::string mutualism_path =
-        config.FILE_PATH() + "SymImpact" + config.FILE_NAME() + file_ending;
-    mutualism_file.open(mutualism_path);
-
-    for (auto pair : dominant_organisms) {
-      auto sample = pair.first.DynamicCast<SGPHost>();
-      if (sample->HasSym()) {
-        mutualism_file
-            << CheckSymbiont(
-                   *sample,
-                   *sample->GetSymbionts().front().DynamicCast<SGPSymbiont>(),
-                   world)
-            << std::endl;
-      } else {
-        // We want something in the file so it overwrites previous data, and NA
-        // is something R generally understands
-        mutualism_file << "NA" << std::endl;
-      }
+  ofstream modularity_file;
```

Comment:
It looks like you got rid of the symbiont impact file code, we need both of those data files